packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -0,0 +1,160 @@
+# Description
+
+This package contains code enabling the production and consumption of typed telemetry for Fluid Framework applications. The typed telemetry from this package is used as the backbone for different Fluid Framework cloud offerings such as dashboards and alarms for Fluid applications. This package can also be used as a reference for customizing and creating their own telemetry solution if desired.
+
+### Telemetry Destinations
+
+At this time, Azure App Insights is the only available destination for Fluid telemetry. Eventually, more destinations will be added.
+
+### What is Application Insights?
+
+At a high level, App Insights is an Azure cloud service for aggregating, visualizing, analyzing, and alerting on metrics related to a given “service” or “application”.
+You create an App Insights Instance and then configure your applications to send data to your instance using either Azure provided SDK’s or REST APIs.
+This could be general machine related health being automatically reported to the instance when you install a logging program on your service’s machines or custom metrics that you manually configure your applications to send. Keep in mind this logger is intended for use with browser based web applications, not pure nodeJS.
+In our case, we are sending custom metrics. [Learn more about Azure App Insights with their docs](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview?tabs=net)
+
+### Areas currently supported for Fluid telemetry
+
+1. Containers
```

Comment:
I'd say not worth trying to document how we're splitting areas when we don't know what other areas, or the future in general, look like.
```suggestion
```

Reply:
I think it's worth a comment somewhere to say this package covers container related telemetry atm. I added a little tidbit to the description section